core/src/main/kotlin/org/jetbrains/kotlinx/dataframe/api/replace.kt

```diff
@@ -53,6 +53,10 @@ public fun <T, C> ReplaceClause<T, C>.with(newColumns: List<AnyCol>): DataFrame<
 }
 
 // TODO: Issue #418: breaks if running on ColumnGroup and its child
+
+/**
+ * @see [Convert.asColumn] for an alternative supported in the compiler plugin
```

Comment:
using `@see` only renders the reference, the text after it is invisible in the IDE :)

Reply:
fixed